softlayer/resource_softlayer_scale_group.go

```diff
@@ -411,9 +435,9 @@ func populateMemberTemplateResourceData(template datatypes.Virtual_Guest) map[st
 	}
 
 	if template.BlockDeviceTemplateGroup != nil {
-		d["block_device_template_group_gid"] = sl.Get(template.BlockDeviceTemplateGroup.GlobalIdentifier)
+		d["image_id"] = sl.Get(template.BlockDeviceTemplateGroup.GlobalIdentifier)
 	} else {
-		d["block_device_template_group_gid"] = nil
+		d["image_id"] = nil
```

Comment:
Ok. I was wondering if that would not be the default already, if there was no key in the map for image_id.